quadrat/block-templates/index.html

```diff
@@ -1,5 +1,9 @@
 <!-- wp:template-part {"slug":"header"} /-->
 
+<!-- wp:spacer {"height":152,"className":"index-spacer"} -->
+<div style="height:152px" aria-hidden="true" class="wp-block-spacer index-spacer"></div>
+<!-- /wp:spacer -->
+
```

Comment:
I used a spacer on the index template because the spacing on the single template is added via the editor using padding on the group block. This way they can both be edited via the editor.